### DISCUSSION TITLE
I need this library to run on python3.5

### DISCUSSION THREAD

**Discussion by NederZweed97** [openai/openai-python#1528] [Q&A]
My NAO robot only supports python3.5. I somehow got the API working several weeks ago and since a few days it stopped working. Are there any possibilities for the API to run on 3.5 (again)?

my backup plan is to use an raspberry pi for communications instead but I wanted to ask this here first before changing plans.

thanks!

**Comment by rattrayalex** [comment]
Sorry, we will not add support for language runtime versions which are EOL.

**Reply by NederZweed97** [reply]
That sucks but understandable
